bukkit/src/main/java/net/william278/huskhomes/hook/RedisEconomyHook.java

```diff
@@ -0,0 +1,25 @@
+package net.william278.huskhomes.hook;
+
+import dev.unnm3d.rediseconomy.api.RedisEconomyAPI;
+import net.william278.huskhomes.HuskHomes;
+import net.william278.huskhomes.HuskHomesInitializationException;
+import org.jetbrains.annotations.NotNull;
+
+public class RedisEconomyHook extends VaultEconomyHook {
+
+
+    public RedisEconomyHook(@NotNull HuskHomes implementor) {
+        super(implementor);
+    }
+
+
+    @Override
+    public boolean initialize() throws HuskHomesInitializationException {
+        RedisEconomyAPI api=RedisEconomyAPI.getAPI();
+        if(api!=null){
+            economy=api.getCurrencyByName(plugin.getSettings().currencyName);
+            return true;
+        }else
+            return super.initialize();
```

Comment:
Not a big fan of the braceless-else blocks, leads to easy mistakes, so another style correction here -- probably can just get rid of the else actually as we're returning true anyway. Just for readability.

Also, get rid of the double spacing between methods and the class signature

Reply:
Done and fixed NPE with non-existent currency